src/meta/topics/dependencies.md

```diff
@@ -0,0 +1,90 @@
+# Dependencies
+
+There are several dependencies that Node.js relies on to work the way it does.
+
+- [Libraries](#Libraries)
+	- [V8](#V8)
+	- [libuv](#libuv)
+	- [http-parser](#http-parser)
+	- [c-ares](#c-ares)
+	- [OpenSSL](#OpenSSL)
+	- [zlib](#zlib)
+- [Tools](#Tools)
+	- [npm](#npm)
+	- [gyp](#gyp)
+	- [gtest](#gtest)
```

Comment:
My preference would be to use two spaces for tabs (vs. tab characters.) Anyone else feel strongly about this either way?